### PR TITLE
445 catch up updates analytics

### PR DESCRIPTION
**Pull request by FayHaifei**

### CODE REVIEW

**Comment by dengyaxi** [comment]
- [ ] 左侧目录需要按 issue 里的样子调整
- [ ] Chat 那两篇内容没有迁移，要加一下，Chat 的目录下应该已经迁移过了

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Not sure what has happened, but when I run this locally all the platform buttons have disappeared. Can you fix it and try again please.

**Comment by dengyaxi** [comment]
There are no platform buttons because Analytics applies to all platforms. Similar to what we have decided for Media Push and Pull.

File: shared/variables/product.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const data = {
 
   },
 
-  'analytics': {
+  'agora-analytics': {
```
[the remark this answers is not in the document]
Comment (dengyaxi) [reply shared/variables/product.js:28]: The previous variable you used does not work. That's why we changed it. It needs to be consistent with the product id that Shuvo defines here: https://github.com/AgoraIO/Docs/blob/main/data/products.js

We've tested it locally and it works fine

[the remark this answers is not in the document]
Comment (billy-the-fish) [reply shared/variables/product.js:28]: :-D

**Review by billy-the-fish — APPROVED** [review]

**Review by dengyaxi — APPROVED** [review]

**Pull request merged**